src/chain/shared_state.rs

```diff
@@ -310,6 +315,10 @@ impl SharedState {
             index
         }
     }
+
+    pub fn increment_section_version(&mut self) {
+        self.section_version = self.section_version.wrapping_add(1);
```

Comment:
will `saturating_add` be more suitable ?

Reply:
I wanted this number to keep changing even when we reach the max value. That would for a brief time break the logic of preferring nodes that joined earlier to those that joined later when selecting who to promote, but that should not be a big problem. That said, this number wraps around after ~1 quintillion churn events, so we won't hit it any time soon :)